save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestView.kt

```diff
@@ -88,20 +96,35 @@ class ContestView : AbstractView<ContestViewProps, ContestViewState>(false) {
                 "#/${FrontendRoutes.CONTESTS.path}/${props.currentContestName}")
         } else if (props.location != prevProps.location) {
             urlAnalysis(ContestMenuBar, Role.NONE, false)
+        } else if (props.currentContestName != prevProps.currentContestName) {
+            getContest()
         }
     }
 
     override fun componentDidMount() {
         super.componentDidMount()
         urlAnalysis(ContestMenuBar, Role.NONE, false)
         getIsFeaturedAndSetState()
+        getContest()
+    }
+
+    private fun getContest() {
```

Comment:
suggestion to rename `getContest` to `fetchContest` for example, because it's not a getter

Reply:
Renamed